### ISSUE TITLE
ktlint incorrectly thinks there should be a space between the lambda and its label

### ISSUE THREAD

**Issue by SUPERCILEX** [shyiko/ktlint#167]
See: https://kotlinlang.org/docs/reference/coding-conventions.html#lambda-formatting

And the code that breaks it:
```kt
private val shouldEjectBlock = block@{ (pathProgress ?: return@block false) >= 0.85 }
```

**Comment by shyiko** [comment]
Hi Alex. 
Current behavior is consistent with Intellij built-it formatter (see  https://github.com/shyiko/ktlint/pull/62#issuecomment-312309584). 
I'd say we hold off until either Intellij is updated to reflect the change or there is a way to configure Intellij built-it formatter not to put space between @ and {.

**Comment by SUPERCILEX** [comment]
@shyiko That's no longer the case (with the 1.2.30 plugin at least), here's a screenshot of IntelliJ complaining:
![image](https://user-images.githubusercontent.com/9490724/36948955-e39dfc28-1f96-11e8-99ef-603f47cce069.png)
And if I run the auto formatter, it attaches the `@` with the bracket.

**Comment by shyiko** [comment]
Alright. In that case I'm merging https://github.com/shyiko/ktlint/pull/62 in and releasing [0.19.0](https://github.com/shyiko/ktlint/releases/tag/0.19.0). It should become available within ~30m.

P.S. Thanks for the PR, @aem!
